src/onegov/api/models.py

```diff
@@ -12,31 +12,46 @@
 from onegov.core.orm import Base
 from onegov.core.orm.types import UUID, UTCDateTime
 from onegov.user import User
+from onegov.core.collection import _M
 
 
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Any, Generic
```

Comment:
```suggestion
from typing import TYPE_CHECKING, Any, Generic, overload
```
Necessary import to make the other suggestion work